{"uid":"177b7eab8ee5d37e","name":"test_list_agent_preferred_phone_10_459","fullName":"tests.test_checklist.TestPromotionChecklist#test_list_agent_preferred_phone_10_459","historyId":"f89e4d1a707d45b40ba2be4209b1b47c","time":{"start":1739798533416,"stop":1739798688333,"duration":154917},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist","time":{"start":1739788254308,"stop":1739788254308,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"passed","steps":[{"name":"Click authorisation button","time":{"start":1739798534232,"stop":1739798534987,"duration":755},"status":"passed","steps":[{"name":"Get authorisation button","time":{"start":1739798534232,"stop":1739798534241,"duration":9},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Source select","time":{"start":1739798535521,"stop":1739798535638,"duration":117},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"source","value":"'459'"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798536989,"stop":1739798537086,"duration":97},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"1"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798537087,"stop":1739798545367,"duration":8280},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"\nMetadata = PROPERTY Ok âœ…\n","time":{"start":1739798545391,"stop":1739798545391,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798545416,"stop":1739798545547,"duration":131},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"2"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798545547,"stop":1739798555567,"duration":10020},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class CUSTOMPROPERTY is not mapped","time":{"start":1739798555568,"stop":1739798555568,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798555604,"stop":1739798555718,"duration":114},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"3"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798555718,"stop":1739798565767,"duration":10049},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class DATASYSTEM is not mapped","time":{"start":1739798565768,"stop":1739798565768,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798565789,"stop":1739798565907,"duration":118},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"4"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798565907,"stop":1739798575964,"duration":10057},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class ENUMERATION is not mapped","time":{"start":1739798575964,"stop":1739798575964,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798575984,"stop":1739798576106,"duration":122},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798576106,"stop":1739798586157,"duration":10051},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYROOMS is not mapped","time":{"start":1739798586158,"stop":1739798586158,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798586180,"stop":1739798586310,"duration":130},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"6"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798586311,"stop":1739798596324,"duration":10013},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYUNITTYPES is not mapped","time":{"start":1739798596325,"stop":1739798596325,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798596348,"stop":1739798596486,"duration":138},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"7"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798596486,"stop":1739798606523,"duration":10037},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYBUILDING is not mapped","time":{"start":1739798606524,"stop":1739798606524,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798606561,"stop":1739798606688,"duration":127},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"8"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798606688,"stop":1739798616722,"duration":10034},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLASTMARKETSALE is not mapped","time":{"start":1739798616723,"stop":1739798616723,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798616757,"stop":1739798616893,"duration":136},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"9"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798616893,"stop":1739798626951,"duration":10058},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLOCATION is not mapped","time":{"start":1739798626952,"stop":1739798626952,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798626979,"stop":1739798627117,"duration":138},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"10"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798627118,"stop":1739798637133,"duration":10015},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIP is not mapped","time":{"start":1739798637134,"stop":1739798637134,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798637169,"stop":1739798637309,"duration":140},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"11"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798637309,"stop":1739798647329,"duration":10020},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIPTRANSFER is not mapped","time":{"start":1739798647329,"stop":1739798647330,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798647366,"stop":1739798647510,"duration":144},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"12"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798647511,"stop":1739798657573,"duration":10062},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYPRIORSALE is not mapped","time":{"start":1739798657574,"stop":1739798657574,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798657610,"stop":1739798657769,"duration":159},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"13"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798657769,"stop":1739798667827,"duration":10058},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSEARCH is not mapped","time":{"start":1739798667828,"stop":1739798667828,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798667861,"stop":1739798668015,"duration":154},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"14"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798668015,"stop":1739798678065,"duration":10050},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSITE is not mapped","time":{"start":1739798678066,"stop":1739798678066,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739798678103,"stop":1739798678253,"duration":150},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"15"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739798678253,"stop":1739798688265,"duration":10012},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55c71a11009a <unknown>\n#1 0x55c719bd78b0 <unknown>\n#2 0x55c719c28b99 <unknown>\n#3 0x55c719c28d21 <unknown>\n#4 0x55c719c77854 <unknown>\n#5 0x55c719c4e9ad <unknown>\n#6 0x55c719c74b8b <unknown>\n#7 0x55c719c4e753 <unknown>\n#8 0x55c719c1a38e <unknown>\n#9 0x55c719c1bb51 <unknown>\n#10 0x55c71a0d976b <unknown>\n#11 0x55c71a0dd6f2 <unknown>\n#12 0x55c71a0c58fc <unknown>\n#13 0x55c71a0de2e4 <unknown>\n#14 0x55c71a0a99cf <unknown>\n#15 0x55c71a0fecd8 <unknown>\n#16 0x55c71a0feeb6 <unknown>\n#17 0x55c71a10ef16 <unknown>\n#18 0x7f656aa9caa4 <unknown>\n#19 0x7f656ab29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYTAXASSESSMENT is not mapped","time":{"start":1739798688266,"stop":1739798688266,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":48,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist::0","time":{"start":1739801612961,"stop":1739801612961,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.filterwarnings('ignore::DeprecationWarning')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_checklist"},{"name":"subSuite","value":"TestPromotionChecklist"},{"name":"host","value":"fv-az1043-579"},{"name":"thread","value":"3351-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_checklist"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.filterwarnings('ignore::DeprecationWarning')"]},"source":"177b7eab8ee5d37e.json","parameterValues":[]}